concreep-refilled/settings.lua

```diff
@@ -1,49 +1,58 @@
 data:extend({

```

Comment:
we should move the new DEFAULT_s above here, and then reference them throughout the data struct below. makes maintenance a bit easier.

Reply:
check